docker/run_worker.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# BBB: Celery 4
```

Comment:
Do you know what BBB stands for? It seems to mean "TODO", and always is used around Celery 4 code.

Reply:
I have no idea. I don't think I've seen BBB in code before.

@hannosch ^^^ What does BBB mean?

Reply:
BBB means “backwards compatibility code”, so things which can be dropped or changed once a dependency upgrade is done or some old Python version no longer needs to be supported. It is used in open-source projects, which also use XXX as a means to signal an urgent todo task / something that must be fixed before the next release. Other projects use TODO, FIXME or other markers.

The Celery 4 stuff is around, as I did the upgrade of all the code to Celery 4 including adjusting to all the changed config settings. But than had to roll back to Celery 3 as version 4 wasn’t production ready at the time. Since these changes were all over the place, I kept them in the code, so it would have been easy to do the upgrade later on. I did expect that to happen a couple weeks later, not years after.